src/app/app.component.html

```diff
@@ -1,3 +1,5 @@
+<div class="fix-container">
```

Comment:
The bottom of the scrollbar is not seen. If you try a resolution of 320x480, the scrollbar is not seen after add work item. 

![309](https://cloud.githubusercontent.com/assets/1892722/20472839/a26c31ea-afe2-11e6-994d-12b23da26cee.png)


Even in higher resolution, the bottom scroll arrow is not seen. Also the last work item is partially hidden.